src/pybind/cephfs.py

```diff
@@ -194,13 +194,33 @@ def require_state(self, *args):
         raise LibCephFSStateError("You cannot perform that operation on a "
                                   "CephFS object in state %s." % (self.state))
 
-    def __init__(self, conf=None, conffile=None):
+    def __init__(self, conf=None, conffile=None, auth_id=None, rados_inst=None):
         self.libcephfs = load_libcephfs()
         self.cluster = c_void_p()
 
+        self.state = "uninitialized"
+        if rados_inst is not None:
+            return self.create_with_rados(rados_inst)
```

Comment:
Should probably check here that the caller hasn't filled in the auth_id (and possibly the conf/conffile) since we're ignoring it in favor of the existing rados inst.